src/settings/controllers/LazyLoadingController.js

```diff
@@ -15,10 +15,15 @@ limitations under the License.
 */
 
 import SettingController from "./SettingController";
-import dis from "../../dispatcher";
+import MatrixClientPeg from "../../MatrixClientPeg";
+import PlatformPeg from "../../PlatformPeg";
 
 export default class LazyLoadingController extends SettingController {
-    onChange(level, roomId, newValue) {
-        dis.dispatch({action: 'flush_storage_reload'});
+    async onChange(level, roomId, newValue) {
```

Comment:
copied the code from `UserSettings` here as I the method there is used in a different use case, to not have this break when the method changes.